runtime/service/src/test/java/org/apache/polaris/service/exception/ExceptionMapperTest.java

```diff
@@ -48,6 +48,7 @@ public class ExceptionMapperTest {
   public void testFullExceptionIsLogged(
       ExceptionMapper<Exception> mapper, Exception exception, Level level) {
     Logger logger = (Logger) LoggerFactory.getLogger(mapper.getClass());
+    logger.setLevel(level);
```

Comment:
Huh? What's the reason for adding this?

Reply:
Yes :-( there are similarly-named inner classes in `OidcTenantConfiguration`.

Reply:
This test si brittle because it assumes that the logger has tested level enabled, but this is not the case anymore.

Anyways, this test is not very Quarkus-friendly. I will refactor it later.